apps/src/code-studio/pd/application/teacher1920/Teacher1920Application.jsx

```diff
@@ -26,8 +26,8 @@ export default class Teacher1920Application extends FormController {
   getPageComponents() {
     return [
       Section1AboutYou,
-      Section2ChooseYourProgram,
       Section3TeachingBackground,
+      Section2ChooseYourProgram,
```

Comment:
Note that I chose to not rename the classes & files for these, despite switching their order, just to reduce churn on the source code.